pkgs/test_api/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.2
```

Comment:
```suggestion
## 0.4.2-dev
```

Lets leave these as `-dev` for right now (below also) - I am going to merge these internally to validate other changes and then we can prep for a release.

Reply:
I updated to `0.4.2-dev` everywhere.